api/tests/routes/backoffice_v3/finance_test.py

```diff
@@ -46,6 +46,41 @@ def test_list_incidents_without_filter(self, authenticated_client):
         assert rows[0]["Type d'incident"] == "Trop Perçu"
 
 
+class GetIncidentCancellationFormTest(GetEndpointHelper):
+    endpoint = "backoffice_v3_web.finance_incident.get_finance_incident_cancellation_form"
+    endpoint_kwargs = {"finance_incident_id": 1}
+    needed_permission = perm_models.Permissions.MANAGE_INCIDENTS
+
+    def test_get_incident_cancellation_form(self, legit_user, authenticated_client):
+        incident = finance_factories.FinanceIncidentFactory()
+        url = url_for(self.endpoint, finance_incident_id=incident.id)
+        response = authenticated_client.get(url)
+
+        assert response.status_code == 200
+
+
+class CancelIncidentTest(PostEndpointHelper):
+    endpoint = "backoffice_v3_web.finance_incident.cancel_finance_incident"
+    endpoint_kwargs = {"finance_incident_id": 1}
+    needed_permission = perm_models.Permissions.MANAGE_INCIDENTS
+
+    def test_cancel_incident(self, authenticated_client):
+        incident = finance_factories.FinanceIncidentFactory()
+        self.form_data = {"comment": "L'incident n'est pas conforme"}
+
+        response = self.post_to_endpoint(authenticated_client, finance_incident_id=incident.id, form=self.form_data)
+
+        assert response.status_code == 200
+
+        created_incidents = finance_models.FinanceIncident.query.filter(
+            finance_models.FinanceIncident.status == finance_models.IncidentStatus.CREATED
+        ).all()
+        assert len(created_incidents) == 0
```

Comment:
Proposition plus courte et ne rapatriant pas les lignes de la bdd :
```suggestion
        assert finance_models.FinanceIncident.query.filter(
            finance_models.FinanceIncident.status == finance_models.IncidentStatus.CREATED
        ).count() == 0
```